Twig/Extension/FormFlowExtension.php

```diff
@@ -36,8 +41,14 @@ public function getFilters() {
 	 * {@inheritDoc}
 	 */
 	public function getFunctions() {
+		if (version_compare(\Twig_Environment::VERSION, '1.12', '<')) {
```

Comment:
I suggest just dropping support for such old versions instead (which are insecure anyway). Given that Twig 1.x is fully BC, there is no reason to stick to such very old (buggy) versions.
Symfony does not support such old versions anymore since a long time btw